api/drivers/qemu/driver.js

```diff
@@ -76,18 +85,21 @@ class QemuDriver extends Driver {
    * @return {Promise[Machine]} The created machine
    */
   createMachine(options) {
-    return ConfigService.get('qemuServiceURL', 'qemuServicePort',
-      'qemuMemory', 'qemuCPU', 'qemuDrive', 'plazaPort',
-      'qemuMachineUsername', 'qemuMachinePassword')
-      .then((config) => {
+    return Promise.props({
+      config: ConfigService.get('qemuServiceURL', 'qemuServicePort',
+        'qemuMemory', 'qemuCPU', 'qemuDrive', 'plazaPort',
+        'qemuMachineUsername', 'qemuMachinePassword'),
+      image: MachineService.getDefaultImage()
+    })
+      .then((resConfigImage) => {
```

Comment:
You should use object destructuring to write this code better: http://www.2ality.com/2015/01/es6-destructuring.html